kernel/pci/src/lib.rs

```diff
@@ -207,6 +216,37 @@ pub fn pci_device_iter() -> Result<impl Iterator<Item = &'static PciDevice>, &'s
     Ok(get_pci_buses()?.iter().flat_map(|b| b.devices.iter()))
 }
 
+static INTX_DEVICES: RwLock<Vec<&'static PciDevice>> = RwLock::new(Vec::new());
+
+// Architecture-independent PCI interrupt handler
+// Aarch64-only because legacy interrupts aren't supported on x86
```

Comment:
just clarifying your comment, which made it sound like x86 itself doesn't support this 
```suggestion
// Currently aarch64-only, because legacy interrupts aren't supported on x86 yet.
```